eppoclient/client_test.go

```diff
@@ -7,6 +7,9 @@ import (
 	"github.com/stretchr/testify/mock"
 )
 
+var defaultAllocationKey = "allocation-key"
```

Comment:
This is just a test variable; doesn't have to be called "allocation-key", could be called something else too.
The SDK uses the `allocationKey` of the matched rule to get an allocation. If there are no rules, the API inserts a default rule (rule with no conditions). Since there are no conditions, the default rule always matches and the SDK uses the allocation corresponding to the default rule's `allocationKey`.